packages/mui-material/src/StepLabel/StepLabel.test.js

```diff
@@ -18,16 +18,11 @@ describe('<StepLabel />', () => {
     render,
     refInstanceof: window.HTMLSpanElement,
     testVariantProps: { error: true },
-    testLegacyComponentsProp: true,
+    testLegacyComponentsProp: false,
```

Comment:
changed to false since this component doesn't have `components` prop

Reply:
Removing this line would have the same effect if I'm not mistaken. If it is the same, then I would prefer to remove it.